examples/pylab_examples/fonts_demo.py

```diff
@@ -13,7 +13,7 @@
 alignment = {'horizontalalignment': 'center', 'verticalalignment': 'baseline'}
 # Show family options
 
-families = ['serif', 'sans-serif', 'cursive', 'fantasy', 'monospace']
+families = ['serif', 'sans-serif', 'monospace']
```

Comment:
Ah.  I removed them just because they give a doc build warning on my machine.  But if it works on Travis, might as well reinstate.  I'll just revert this change.